api-operator/deploy/controller-configs/api_controller_conf.yaml

```diff
@@ -84,7 +84,7 @@ kind: ConfigMap
 metadata:
   name: envoy-mgw-configs
 data:
-  mgwAdapterHost: "https://adapter.default:9843"
+  mgwAdapterHost: "https://adapter-service:9843"
```

Comment:
From MG m4 release we changed the name of the adapter service to `adapter` to align with docker-compose service name
https://github.com/wso2/product-microgateway/pull/1574/files#diff-7e2af33631f6c25b77eb84c52fccc822592cfd958dc435d4994e5164f9a8ca84

Reply:
But still, we support M3 and this is the working config.

Reply:
go it